src/Build/System.Text.Encodings.Web.pkgdef

```diff
@@ -3,5 +3,5 @@
 "codeBase"="$BaseInstallDir$\MSBuild\Current\Bin\System.Text.Encodings.Web.dll"
 "publicKeyToken"="cc7b13ffcd2ddd51"
 "culture"="neutral"
-"oldVersion"="0.0.0.0-4.0.5.0"
-"newVersion"="4.0.5.0"
+"oldVersion"="0.0.0.0-5.0.0.1"
+"newVersion"="5.0.0.1"
```

Comment:
This whole file should be deleted if our direction is that VS should have its own copy of this file, since MSBuild should then not be telling VS to look at the msbuild copy.